chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomCustomDataSpek.kt

```diff
@@ -0,0 +1,177 @@
+package com.pusher.chatkit
+
+import com.google.common.truth.Truth.assertThat
+import com.pusher.chatkit.Rooms.GENERAL
+import com.pusher.chatkit.Rooms.SAMPLE_CUSTOM_DATA
+import com.pusher.chatkit.Users.ALICE
+import com.pusher.chatkit.Users.PUSHERINO
+import com.pusher.chatkit.Users.SUPER_USER
+import com.pusher.chatkit.test.InstanceActions.createDefaultRole
+import com.pusher.chatkit.test.InstanceActions.newRoom
+import com.pusher.chatkit.test.InstanceActions.newUsers
+import com.pusher.chatkit.test.InstanceSupervisor.setUpInstanceWith
+import com.pusher.chatkit.test.InstanceSupervisor.tearDownInstance
+import com.pusher.chatkit.util.FutureValue
+import junit.framework.Assert.assertNotNull
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+
+object RoomCustomDataSpek : Spek({
```

Comment:
For new tests I was using clear naming. How does `RoomCustomDataE2eTest` sound? I found it easier to navigate when tests are clearly named by their type and actually called test instead of naming them after the framework used (plus Spec vs Spek — for actual names in the docs they use `c`).

Reply:
Hmm. I'm not sure how I feel about having `E2e`. The other tests are currently `*Spek` so I don't want to add too much confusion there. Let's talk about this for v2 on what kind of naming schemes we want for to use and keep so we can be as consistent?

Reply:
The existing tests were named as "Spec", "Spek" or "Test". I added quite a few named as "Test".

I think that `E2e` is the most descriptive name. I was also thinking about "System" but thought that it's better to keep that name for OS integration tests (UI Automator) if we create any in the future. "End to end" also conveys the information more clearly than "system".

Fair enough if you are not convinced and don't feel like changing it now ;) I just wanted to use this opportunity for that new(ish) test to be named consistently with `FunctionalTest`s that has been recently created.

Reply:
Have compromised and renamed it to `Test` :) We can talk about what kind of naming convention we want for v2 and start renaming things 😄